infra/aws/terraform/management-account/organization-accounts-boskos.tf

```diff
@@ -204,3 +204,32 @@ module "k8s_infra_eks_e2e_boskos_010" {
     "service"     = "boskos"
   }
 }
+
+
+module "k8s_infra_e2e_boskos_scale_001" {
+  source = "../modules/org-account"
+
+  account_name               = "k8s-infra-e2e-boskos-scale-001"
```

Comment:
I think we should call these accounts similar to boskos accounts, i.e. `k8s-infra-eks-e2e-boskos-scale-001`. This is considering these accounts are used by boskos in the EKS cluster.

Reply:
Not necessarily. The accounts could used on a different boskos config running on a non-EKS cluster (kops cluster, fargate, etc...).
We currently use static credentials for authn between Boskos and the accounts. That give the flexibility to reuse on a different environment.
I prefer to keep them generic.